src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2119,8 +2119,17 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public System.String ToUpper(System.Globalization.CultureInfo culture) { throw null; }
         public string ToUpperInvariant() { throw null; }
         public string Trim() { throw null; }
+#if netcoreapp11
```

Comment:
Correct we no longer need these #ifdef's. @joperezr was cleaning some of these up but might have missed this project. @justinvp can you please go ahead and remove these #ifdefs for this project?